bitcraze_crazyflie_2/cf2.xml

```diff
@@ -0,0 +1,119 @@
+<mujoco model="cf2">
+  <option integrator="RK4"/>
```

Comment:
Is RK4 necessary?

Reply:
Using RK4 is not necessary here, but i feel its a better choice. Considering the small inertial properties, if any down wash models or external disturbances are implemented, then a stable integrator might provide a more accurate simulation.